cmd/neofs-cli/modules/object/util.go

```diff
@@ -250,13 +245,7 @@ func OpenSession(ctx context.Context, cmd *cobra.Command, dst SessionPrm, key *e
 //
 // If provided SessionPrm is of type internal.DeleteObjectPrm, OpenSessionViaClient
```

Comment:
i read https://github.com/nspcc-dev/neofs-node/pull/2875/files#r1655445343, then noticed this place, and wanted to clarify that input could be much smaller for big objects. But this is irrelevant since e97444aa743322da7876441b1b95c563fc902b10. Let drop, and pls more attention to docs